### ISSUE TITLE
Execution failed for task ':app:preDebugBuild'.

### ISSUE THREAD

**Issue by hossameldinmi** [googlecodelabs/photos-sharing#3]
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':app:preDebugBuild'.
> Android dependency 'androidx.core:core' has different version for the compile (1.0.0) and runtime (1.0.2) classpath. You should manually set the same version via DependencyResolution

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 11s
*******************************************************************************************
The Gradle failure may have been because of AndroidX incompatibilities in this Flutter app.
See https://goo.gl/CP92wY for more information on the problem and how to fix it.
*******************************************************************************************
Gradle task assembleDebug failed with exit code 1

**Comment by llleodeleon** [comment]
I have the same issue @cassierecher @jfschmakeit

**Comment by davidglivar** [comment]
https://github.com/flutter/flutter/issues/27254#issuecomment-468526704

upgrading gradle resolved this for me

**Comment by hossameldinmi** [comment]
I got the solution 
It's related to **Android X** Compatibility, read this article 
[Migrating to Android X](https://flutter.dev/docs/development/packages-and-plugins/androidx-compatibility) for **Migrating to Android X** and this that solves the problem.

**Comment by jfschmakeit** [comment]
Just fixed this in  cc6e09a - thanks for the pointer!

**Comment by panicoli0** [comment]
Only need to add the following code on the build.gradle file:

```
configurations.all {
    resolutionStrategy {
        force 'androidx.media:media:1.0.0'
    }
}
```

Then, the app runs without any error.
